gateway/api/models.py

```diff
@@ -190,6 +213,9 @@ class Job(models.Model):
         choices=JOB_STATUSES,
         default=QUEUED,
     )
+    sub_status = models.CharField(
+        max_length=30, choices=SUB_STATUSES, default=None, null=True, blank=True
```

Comment:
@korgan00 how do you see to configure `max_length` as 255. I don't think there is a real reason for that 30 and we can avoid future problems in case we want a greater value.

Reply:
I just took the longest one and rounded up :)
I'm ok with 255.